Gemfile

```diff
@@ -71,7 +71,7 @@ end
 # Use rspec+selenium_webdriver+capybara as for test
 group :test do
   gem 'capybara', '~> 2.2'
-  gem 'codeclimate-test-reporter', '~> 1.0.0'
+  # gem 'codeclimate-test-reporter', '~> 1.0.0' # Removed: Code Climate API deprecated (July 2025)
```

Comment:
Please remove the commented string from the Gemfile - it does not make any sense to keep it there.